sdk/communication/azure-communication-rooms/src/test/java/com/azure/communication/rooms/RoomsTestBase.java

```diff
@@ -39,24 +39,27 @@ public class RoomsTestBase extends TestBase {
     protected static final TestMode TEST_MODE = initializeTestMode();
 
     protected static final String CONNECTION_STRING = Configuration.getGlobalConfiguration().get(
-                    "COMMUNICATION_CONNECTION_STRING_ROOMS",
-                "endpoint=https://REDACTED.communication.azure.com/;accesskey=P2tP5RwZVFcJa3sfJvHEmGaKbemSAw2e");
+            "COMMUNICATION_CONNECTION_STRING_ROOMS",
+            "endpoint=https://REDACTED.communication.azure.com/;accesskey=P2tP5RwZVFcJa3sfJvHEmGaKbemSAw2e");
 
     protected static final OffsetDateTime VALID_FROM = OffsetDateTime.now();
     protected static final OffsetDateTime VALID_UNTIL = VALID_FROM.plusDays(30);
 
-    protected static RoomJoinPolicy roomJoinPolicy = RoomJoinPolicy.INVITE_ONLY;
-
     protected List<RoomParticipant> participants1;
```

Comment:
All these participants defined here make the tests really hard to read - readers have to jump back and forth between test file and this file to figure out what is being tested. I would suggest refactoring and defining everything inside RoomsAsyncClientTests. You can look at the .NET tests as reference. I'm also okay with not doing it in this PR and tracking the work as cleanup for later. If you want to take this approach, please link the ADO item here to track this work.

Reply:
I still kept the same structure but cleaned up everything. It should be easy to follow now.